### PR TITLE
Validation updates

### PR DESCRIPTION
**Pull request by jrichardBD**
autoImportHttpsCertificates is false by default. The integrations will have to expose a way for Users to configure the import

### CODE REVIEW

File: src/main/java/com/blackducksoftware/integration/hub/builder/HubServerConfigBuilder.java [hunk 1]
```diff
@@ -62,11 +68,46 @@
...
     @Override
+    public HubServerConfig build() throws IllegalStateException {
+        try {
+            return super.build();
+        } catch (final IllegalStateException e) {
+            if (e.getMessage().contains("SunCertPathBuilderException")) {
+                if (autoImportHttpsCertificates) {
+                    final HubCertificateHandler handler = new HubCertificateHandler(getLogger());
+                    try {
+                        if (StringUtils.isNotBlank(hubUrl)) {
```
Comment (psantos1113) [inline src/main/java/com/blackducksoftware/integration/hub/builder/HubServerConfigBuilder.java]: Is it possible for the hub url to be empty at this point?  if we get the SunCertPathBuilderException we tried to connect to a hub url correct?

Comment (psantos1113) [inline src/main/java/com/blackducksoftware/integration/hub/builder/HubServerConfigBuilder.java]: Spoke to @jrichardBD  this is defensive coding.  Not worth holding up the pull request over this.

Comment (jrichardBD) [inline src/main/java/com/blackducksoftware/integration/hub/builder/HubServerConfigBuilder.java]: We can probably make some assumptions or we can add more defensive code. Not sure what the team wants to do

**Comment by jrichardBD** [comment]
Still making changes, so I removed the reviewers that I could until I have everything finalized

**Comment by jrichardBD** [comment]
Please re-review these changes

**Pull request merged**